<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configDialog</class>
 <widget class="QDialog" name="configDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="configTabWidget">
     <property name="currentIndex">
      <number>10</number>
     </property>
     <widget class="QWidget" name="operatorTab">
      <attribute name="title">
       <string>Operator</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="operatorConfig" name="operatorWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="directoryTab">
      <attribute name="title">
       <string>Directories</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="directoriesConfig" name="directoryWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="guiTab">
      <attribute name="title">
       <string>Gui</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="guiConfig" name="guiWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="soundTab">
      <attribute name="title">
       <string>Sound</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="soundConfig" name="soundWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="catTab">
      <attribute name="title">
       <string>CAT</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="rigConfig" name="catWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cwTab">
      <attribute name="title">
       <string>CW</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="cwConfig" name="cwWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="repeaterTab">
      <attribute name="title">
       <string>Repeater</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="repeaterConfig" name="repeaterWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ftpTab">
      <attribute name="title">
       <string>FTP</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="ftpConfig" name="ftpWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="waterfallTab">
      <attribute name="title">
       <string>Waterfall</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="waterfallConfig" name="waterfallWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="hybridTab">
      <attribute name="title">
       <string>Hybrid</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="hybridConfig" name="hybridWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="profilesTab">
      <attribute name="title">
       <string>DRM Profiles</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="drmProfileConfig" name="drmProfilesWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="frequenciesTab">
      <attribute name="title">
       <string>Frequencies</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="frequencySelectWidget" name="freqSelectWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>soundConfig</class>
   <extends>QWidget</extends>
   <header>soundconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>directoriesConfig</class>
   <extends>QWidget</extends>
   <header>directoriesconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>operatorConfig</class>
   <extends>QWidget</extends>
   <header>operatorconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>guiConfig</class>
   <extends>QWidget</extends>
   <header>guiconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>rigConfig</class>
   <extends>QWidget</extends>
   <header>rigconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cwConfig</class>
   <extends>QWidget</extends>
   <header>cwconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>repeaterConfig</class>
   <extends>QWidget</extends>
   <header>repeaterconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ftpConfig</class>
   <extends>QWidget</extends>
   <header>ftpconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>waterfallConfig</class>
   <extends>QWidget</extends>
   <header>waterfallconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>hybridConfig</class>
   <extends>QWidget</extends>
   <header>hybridconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>drmProfileConfig</class>
   <extends>QWidget</extends>
   <header>drmprofileconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>frequencySelectWidget</class>
   <extends>QWidget</extends>
   <header>frequencyselectwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>configDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>configDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
